contributing/google-season-of-docs-2019/README.md

```diff
@@ -35,12 +35,21 @@ If you have questions and prefer email, please write us at:  gsod+2019@rocket.ch
 
 ## Project Ideas
 
-### Application Programming Interfaces (APIs) Documentation Playground
+### Application Programming Interfaces (APIs) Documentation Playground (GSoC follow up)
 
 - **Mentors**: @martin.schoeler, @marcos.defendi
-- **Description**: When developing applications, developers can use a rich set of Application Programming Interfaces (APIs) offered by Rocket.Chat to control every aspect of Rocket.Chat's operation. The documentation for this set of APIs mixes automated generation with hand crafted usage descriptions, application scenarios, and coding examples. The "playground" makes this documentation comes alive.  Developer can visit the API page and actually interact with the API directly through the documentation page. We already have a hand-crafted set of API docs that is maintained by our community but they tend to become obsolete as API changes. This project will replace the legacy documentation.  You will work with two mentors specialized in API coding and documentatation generation for this project.
+- **Description**: When developing applications, developers can use a rich set of Application Programming Interfaces (APIs) offered by Rocket.Chat to control every aspect of Rocket.Chat's operation. The documentation for this set of APIs mixes automated generation with hand crafted usage descriptions, application scenarios, and coding examples. The "playground" makes this documentation comes alive.  Developer can visit the API page and actually interact with the API directly through the documentation page. We already have a hand-crafted set of API docs that is maintained by our community but they tend to become obsolete as API changes. This project will replace the legacy documentation. You will work with two mentors specialized in API coding and documentation generation for this project.
+- In this project, we will **not** be creating the playground or the docs generation systems, that will be handled by the Google Summer of Code student ([See the GSoC Project]()). Initially some of the endpoints will not be able to be on the playground or generated. The objective of this project is to find a way to add these missing endpoints to the playground system and find ways to improve our api documentation.
```

Comment:
Please add the URL.    

Also, Google Summer of Code will finish before GSoD officially starts, and there is no need to mention this.  I think we need to assume that the playground is completed and in place by the time they start working.   

Also don't think the writers should be doing ANY programming or adding endpoints.   They would just be writing the documentation and/or perhaps help in designing the "structure" of the docs.